.github/workflows/pr.yaml

```diff
@@ -71,9 +71,8 @@ jobs:
           java-version: 11
       - name: Test
         uses: gradle/gradle-build-action@v2
-        env:
-          GRADLE_BUILD_ACTION_CACHE_KEY_PREFIX: ${{github.workflow}}#${{github.run_number}}:${{github.run_attempt}}-
         with:
+          cache-disabled: true
```

Comment:
the last CI failure was a sporadic `MicrometerTest` failure: https://github.com/microsoft/ApplicationInsights-Java/runs/6619718932?check_suite_focus=true

Reply:
GRADLE_BUILD_ACTION_CACHE_KEY_PREFIX doesn't seem to work.. it still restores from cache. `cache-disabled: true` seems to work. i no longer find restoring from cache.

Reply:
this wasn't so bad, took 26 mins.

![image](https://user-images.githubusercontent.com/56097766/170761995-3a031d8d-fcd4-4807-9b51-4348a0e8bd07.png)

we can re-evaluate it later if we need to remove this.